src/multirust-cli/common.rs

```diff
@@ -0,0 +1,134 @@
+//! Just a dumping ground for cli stuff
+
+use multirust::{Cfg, Result, Notification, Toolchain};
+use multirust_utils::{self, utils};
+use multirust_utils::notify::NotificationLevel;
+use std::ffi::OsStr;
+use std::io::{Write, BufRead};
+use std::process::{self, Command};
+use std;
+use tty;
+use term;
+
+pub fn confirm(question: &str, default: bool) -> bool {
+    print!("{} ", question);
+    let _ = std::io::stdout().flush();
+    let input = read_line();
+
+    let r = match &*input {
+        "y" | "Y" => true,
+        "n" | "N" => false,
+        "" => default,
+        _ => false,
+    };
+
+    println!("");
+
+    r
+}
+
+fn read_line() -> String {
+    let stdin = std::io::stdin();
+    let stdin = stdin.lock();
+    let mut lines = stdin.lines();
+    lines.next().unwrap().unwrap()
```

Comment:
Maybe worth putting a TODO here to handle this error eventually? Seems like something that'll definitely be tripped at some point